android/MapboxGLAndroidSDK/src/main/java/com/mapbox/mapboxsdk/views/MapView.java

```diff
@@ -2426,19 +2475,24 @@ public boolean onSingleTapConfirmed(MotionEvent e) {
                 Collections.sort(nearbyMarkers);
 
                 if (nearbyMarkers == mMarkersNearLastTap) {
+
+                    // TODO: We still need to adapt this logic to the new mSelectedMarkers list,
```

Comment:
My bad. Just ticketed it: https://github.com/mapbox/mapbox-gl-native/issues/3156